src/utils/prisma.ts

```diff
@@ -14,3 +14,19 @@ export const prisma =
     });
 
 if (process.env.NODE_ENV !== 'production') global.prisma = prisma;
+
+prisma.$use(async (params: Prisma.MiddlewareParams, next: (params: Prisma.MiddlewareParams) => Promise<any>) => {
```

Comment:
Add some comment with description about this monkey patch if it possible. Great if you have some links to same issues.